xbmc/LangInfo.cpp

```diff
@@ -606,7 +606,7 @@ bool CLangInfo::UseLocaleCollation()
   return m_collationtype == 2;
 }
 
-void CLangInfo::LoadTokens(const TiXmlNode* pTokens, std::set<std::string>& vecTokens)
+void CLangInfo::LoadTokens(const TiXmlNode* pTokens, Tokens& vecTokens)
```

Comment:
nitpick: apparently `vecTokens` seems to be not a std::vector, so shouldnt this be called `tokensSet` ?

Reply:
If we change this, the new name should be agnostic to the container used for implementation, but this is nothing that is in scope of this PR.